drivers/modem/gsm_ppp.c

```diff
@@ -60,6 +60,8 @@ static struct gsm_modem {
 	const struct device *at_dev;
```

Comment:
Please squash the commit "drivers: modem: gsm_ppp: store net_if in context struct" into some existing commit as there is no need to have a separate commit for this change.